### DISCUSSION TITLE
what shape should this have : "FieldName.PAST_FEAT_DYNAMIC_REAL = 'past_feat_dynamic_real'",

### DISCUSSION THREAD

**Discussion by pratikgehlott** [awslabs/gluonts#1729] [Q&A]
Most of the other field's transformation values are listed in the documentation, but this FieldName's transformation is not given.
For example something like this: 

```
feat_static_cat: static (over time) categorical features, list with dimension equal to the number of features

feat_static_real: static (over time) real features, list with dimension equal to the number of features

feat_dynamic_cat: dynamic (over time) categorical features, array with shape equal to (number of features, target length)

feat_dynamic_real: dynamic (over time) real features, array with shape equal to (number of features, target length)
```

`PS - we are using the different lengths of each Time-Series, is this the issue? because when I run using the same length for all the Time-Series the error does not come.`

@mbohlkeschneider, @lostella can you guys help?

**Comment by sayedathar11** [comment]
@lostella @mbohlkeschneider . I am encountering same issue , can you please help with this issue .
